pkg/generators/openapi.go

```diff
@@ -126,7 +128,11 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 			PackagePath: arguments.OutputPackagePath,
 			HeaderText:  header,
 			GeneratorFunc: func(c *generator.Context) (generators []generator.Generator) {
-				return []generator.Generator{NewOpenAPIGen(arguments.OutputFileBaseName, arguments.OutputPackagePath, context)}
+				var reportFilename string
+				if customArgs, ok := arguments.CustomArgs.(*CustomArgs); ok {
+					reportFilename = customArgs.ReportFilename
+				}
```

Comment:
seems like these four lines can be done earlier, before the `return ...` line, rather than repeatedly each time the generator func is called?